support-frontend/assets/helpers/urls/routes.ts

```diff
@@ -71,8 +70,10 @@ function paperSubsUrl(
 		.map((keyValuePair) => keyValuePair.join('='))
 		.join('&');
 
+	const hash = withDelivery ? `#${HomeDelivery}` : '';
```

Comment:
is this still necessary if the page always shows home delivery first anyway?

Reply:
Good shout, I have decided to make the function more explicit that you can either send it a `PaperFulfilmentOptions` which links to the tabs, or `undefined`, which just links to the page.

Reply:
in a future PR? (or did I miss something sorry)

Reply:
Nope, [I missed something](https://github.com/guardian/support-frontend/pull/5572/commits/0fc371a6b09e7dfde4a6cfbef85f563f9165ae54) (forgot to push).